source/resources/postman/Bambora.postman_collection.json

```diff
@@ -5437,6 +5436,79 @@
+									"postman.setEnvironmentVariable(\"passcode_encoded\", passcode_encoded);\r",
+									"\r",
+									"pm.environment.set(\r",
+									"    'transaction_payload',\r",
+									"    JSON.stringify(environment[\"google_pay_transaction_payload\"])\r",
+									");"
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"tests[\"Status code is 200\"] = responseCode.code === 200;\r",
+									"postman.clearEnvironmentVariable(\"passcode_encoded\");\r",
+									"postman.clearEnvironmentVariable(\"transaction_payload\");\r",
+									"\r",
+									"if(responseCode.code === 200) {\r",
+									"    var jsonData = JSON.parse(responseBody);\r",
+									"    // transaction id used for void: \r",
+									"    postman.setEnvironmentVariable(\"transId\", jsonData.id);\r",
+									"}\r",
+									""
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Authorization",
+								"value": "Passcode {{passcode_encoded}}",
+								"type": "text"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\r\n    \"amount\": 10,\r\n    \"payment_method\":\"google_pay\",\r\n    \"customer_ip\": \"123.123.123.123\",\r\n    \"term_url\":\"https://test01-web.na.bambora.com\",\r\n    \"google_pay\":\r\n    {\r\n      \"name\": \"Test User\", \r\n      \"transaction_payload\": {{transaction_payload}},\r\n      \"complete\": true,\r\n      \"3d_secure\": \r\n        {\r\n            \"browser\":\r\n            {\r\n                \"accept_header\": \"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3\",\r\n                \"java_enabled\": \"false\",\r\n                \"language\": \"en-US\",\r\n                \"color_depth\": \"24\",\r\n                \"screen_height\": 1080,\r\n                \"screen_width\": 1920,\r\n                \"time_zone\": -120,\r\n                \"user_agent\": \"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.90 Safari/537.36\",\r\n                \"Javascript_enabled\": true\r\n            },\r\n         \"enabled\": true,\r\n         \"version\": 2,\r\n         \"auth_required\": false\r\n        }\r\n    },\r\n    \"billing\":{\r\n        \"name\": \"Test User\",\r\n        \"address_line1\": \"1755 W. BROADWAY SUITE 105\",\r\n        \"address_line2\": \"\",\r\n        \"city\": \"Victoria\",\r\n        \"province\": \"BC\",\r\n        \"country\": \"CA\",\r\n        \"postal_code\": \"J0L1T0\",\r\n        \"phone_number\": \"1234567890\",\r\n        \"email_address\": \"tester@test.com\"\r\n    }\r\n}"
```

Comment:
How is transaction payload obtained to test this transaction ? Is this saved in bamboo

Reply:
The user of the request will add the payload to the empty environment variable

Reply:
OK , is that documented or clear to the one who uses it ? I understand if existing don't have it but we should start documented or adding comments to indicate how to use this postman collection.

Reply:
I believe they would learn or understand from the Dev Docs guide, @Darren-Worldline can you comment on this?

Reply:
I ran the suggestion by Mark since Integrations & Customer Services are pretty much the owners of this collection. He agrees with Sagar's suggestion. @jwebster-worldline Please add a comment in the body of the request directing the developer/ merchant to supply the payload in the "google_pay_transaction_payload" environment variable.

Reply:
Ok sure, will do

Reply:
Updated